### DISCUSSION TITLE
Desafio Frontend

### DISCUSSION THREAD

**Discussion by Darlan0307** [community/community#118342] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Acabei de concluir um desafio de codificação front-end do [FrontendMentor!](https://www.frontendmentor.io/home) 🎉

Você pode ver minha solução aqui: [solution]( https://www.frontendmentor.io/solutions/rest-countries-api-with-color-theme-switcher--5Tjc0PlBj)

Qualquer sugestão sobre como posso melhorar é bem-vinda!

**Comment by LiteBrite82** [comment]
Hi @Darlan0307 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
